Search devices through SSDP, add basic support to Wemo Insight

luca776 opened the pull request:
Instead of maintaining the list of MAC addresses, the search is performed through the SSDP protocol.
Insight class add support to "standby?" state and can determine the power consumption of the device.

gisikw commented:
Thanks for the PR! I unfortunately don't have devices to quickly test with, and it's been a while since I've used this. I have a vague recollection of running into problems with SSDP when running on JRuby. If it's not too much trouble, mind verifying that this works in that environment?

gisikw commented on wemote.gemspec:
```diff
@@ -18,4 +18,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
+
+  spec.add_dependency 'ssdp'
```
Can we add a version here please? Thanks!

luca776 commented:
Ok, you were right, there was an issue with JRuby, I fixed it on SSDP and the author was very fast to merge my Pull Request. Now I added the dependency to the new gem version (>=1.1.6) and I removed some duplicated code on Insight sublclass.

luca776 commented:
@gisikw do you think it's okay?

gisikw commented:
So sorry for the delay on this. Thanks for getting it working with JRuby - excited not to be shelling out :)
